.github/ISSUE_TEMPLATE/Security_Issue_Report.md

```diff
@@ -1,20 +0,0 @@
----
-name: Security Issue
```

Comment:
Please don't delete this.  It makes sure that I get assigned security issues right away

Reply:
Also, they are allowed to file issues, if MSRC says to, and I'd prefer they have a template to do that.

Reply:
Please answer my comments

Reply:
Will add back